Tests/test_execfile.py

```diff
@@ -13,32 +13,31 @@
 #
 #####################################################################################
 
-from iptest.assert_util import *
-from iptest.file_util import path_combine
-
-
 import os
-
-
-@skip("win32")
-def test_sanity():
-    root = testpath.public_testdir
-    execfile(path_combine(root, "Inc", "toexec.py"))
-    execfile(path_combine(root, "Inc", "toexec.py"))
-    #execfile(root + "/doc.py")
-    execfile(path_combine(root, "Inc", "toexec.py"))
-
-def test_negative():
-    AssertError(TypeError, execfile, None) # arg must be string
-    AssertError(TypeError, execfile, [])
-    AssertError(TypeError, execfile, 1)
-    AssertError(TypeError, execfile, "somefile", "")
-
-def test_scope():
-    root = testpath.public_testdir
-    z = 10
-    execfile(path_combine(root, "Inc", "execfile_scope.py"))
+import unittest
+
+from iptest import IronPythonTestCase, is_cli
+
+@unittest.skipUnless(is_cli, 'IronPython specific test case')
```

Comment:
The skip was only on test_sanity. Although unless there's something cli specific in toexec.py it might not be necessary.